examples/microprofile/graphql/README.md

```diff
@@ -74,6 +74,44 @@ Access the `/graphql` endpoint via `http://127.0.0.1:7001/graphql`:
     {"data":{"createTask":{"id":"0d4a8d","description":"Task Description 1","createdAt":1605501774877,"completed":false}}
     ```  
 
+## Accessing Metrics
+
+In [TaskApi.java](src/main/java/io/helidon/examples/graphql/basics/TaskApi.java), the [Microprofile Metrics](https://github.com/eclipse/microprofile-metrics)
```

Comment:
`@SimplyTimed` may be easier to showcase, as it only has two values. Also you can show the JSON output, which is easier to read for humans (request endpoint with `Accept: application/json`)